paddle/operators/roi_pool_op.cu

```diff
@@ -0,0 +1,265 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/platform/cuda_helper.h"
+#include "paddle/operators/roi_pool_op.h"
+
+namespace paddle {
+namespace operators {
+
+#define FLT_MAX __FLT_MAX__
+
+constexpr int PADDLE_OPERATORS_ROIPOOL_CUDA_NUM_THREADS = 512;
+constexpr int PADDLE_OPERATORS_ROIPOOL_MAXIMUM_NUM_BLOCKS = 4096;
+
+inline int PADDLE_OPERATORS_ROIPOOL_GET_BLOCKS(const int N) {
```

Comment:
I understand that you might want to mark this function returning a constant value, but according to the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Function_Names), we should name functions something like

```c++
static inline int NumBlocks(const int N) {
```